### ISSUE TITLE
Bot mules after each game

### ISSUE THREAD

**Issue by yolo1111** [kolton/d2bot-with-kolbot#2587]
Hi,

I tried every usedInventoryTrigger command:

usedStashTrigger: 80,

//usedInventoryTrigger: 50,
usedInventoryTrigger: 80,
usedInventoryTrigger: 100,
usedInventoryTrigger: 50,
usedInventoryTrigger: 0,

and keeps doing. Every game it mules all stash items even if he tooks very few items. How I can fix it? Thanks

**Comment by Rakkah** [comment]
Post your AutoMule.js (ofcourse without pass/login)
Config.Inventory of  your character could be useful aswell

**Comment by yolo1111** [comment]
Don't know why but think now works with this line:

//usedInventoryTrigger: 50,

**Comment by yolo1111** [comment]
> Post your AutoMule.js (ofcourse without pass/login)
> Config.Inventory of your character could be useful aswell

Config.Inventory[0] = [1,1,1,1,0,0,0,0,0,0];
	Config.Inventory[1] = [1,1,1,1,0,0,0,0,0,0];
	Config.Inventory[2] = [1,1,1,1,0,0,0,0,0,0];
	Config.Inventory[3] = [1,1,1,1,0,0,0,0,0,0];

var AutoMule = {
	Mules: {
		"Mule1": {
			muleProfile: "mule", // The name of mule profile in d2bot#. It will be started and stopped when needed.
			accountPrefix: "xxx", // Account prefix. Numbers added automatically when making accounts.
			accountPassword: "xxx", // Account password.
			charPrefix: "xxx", // Character prefix. Suffix added automatically when making characters.
			realm: "europe", // Available options: "useast", "uswest", "europe", "asia"
			expansion: true,
			ladder: true,
			hardcore: true,
			charsPerAcc: 8, // Maximum number of mules to create per account (between 1 to 18)

			// Game name and password of the mule game. Never use the same game name as for mule logger.
			muleGameName: ["xxx", "xxx"], // ["gamename", "password"]

			// List of profiles that will mule items. Example: enabledProfiles: ["profile 1", "profile 2"],
			enabledProfiles: ["yolo"],

			// Stop a profile prior to muling. Useful when running 8 bots without proxies.
			stopProfile: "",
			stopProfileKeyRelease: false, // true = stopProfile key will get released on stop. useful when using 100% of your keys for botting.

			// Trigger muling at the end of a game if used space in stash and inventory is equal to or more than given percent.
			usedStashTrigger: 80,
			//usedInventoryTrigger: 50,

			// Mule items that have been stashed at some point but are no longer in pickit.
			muleOrphans: true
		}
	},

**Comment by Deviter** [comment]
// Trigger muling at the end of a game if used space in stash and inventory is equal to or more than given percent.
			usedStashTrigger: 80,
			usedInventoryTrigger: 80,

change this

the 

//usedInventoryTrigger: 50,  is bad configuration

**Comment by yolo1111** [comment]
> // Trigger muling at the end of a game if used space in stash and inventory is equal to or more than given percent.
> usedStashTrigger: 80,
> usedInventoryTrigger: 80,
> 
> change this
> 
> the
> 
> //usedInventoryTrigger: 50, is bad configuration

yes but with 80 still muling after each game. i got this inventory but bot keeps 8 potions:

Config.Inventory[0] = [1,1,1,1,0,0,0,0,0,0];
Config.Inventory[1] = [1,1,1,1,0,0,0,0,0,0];
Config.Inventory[2] = [1,1,1,1,0,0,0,0,0,0];
Config.Inventory[3] = [1,1,1,1,0,0,0,0,0,0];

maybe between 90-100 would be good but not 100 because still muling :/

**Comment by yolo1111** [comment]
Still muling with 'usedInventoryTrigger: 90'

I'm very tired.

**Comment by yolo1111** [comment]
UPDATE: both set to 90 and looks good

**Comment by yolo1111** [comment]
well finally i deleted two inventory lines and now works. maybe the bot get bugged when you uses many lines
